packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -64,104 +73,122 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
   )
 
   const PublicationCard = ({ pub }: PublicationCardProps) => (
-    <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+    <Grid
+      item
+      key={pub.rowId}
+      height={{ xs: 'auto', sm: '275px' }}
+      minWidth={{ xs: '280px', lg: 'initial' }}
+      maxWidth={{ xs: '450px', lg: 'initial' }}
+      sx={{
+        paddingTop: '0px !important',
+        paddingLeft: '0px !important',
+      }}
+    >
```

Comment:
I think you should consider switching to [Unstable_GridV2](https://v5.mui.com/material-ui/migration/migration-grid-v2/) if you're having padding problems--this will give us a smoother migration to MUI 6.

If Grid2 does not solve these problems, feel free to ignore this comment.